app/views/releases/index.html.erb

```diff
@@ -1,7 +1,16 @@
 <h1>Releases</h1>
 
 <div class="releases">
+  <% current_year_and_week_number = "none" %>
+
   <% @albums.each do |album| %>
+    <!-- Does this album fall into a new week? -->
+    <% album_release_date = Date.parse(album["release_date"]).to_date %>
+    <% release_year_week = "#{album_release_date.year} Week #{album_release_date.cweek}" %>
+    <% if current_year_and_week_number != release_year_week  %>
+      <% current_year_and_week_number = release_year_week %>
+      <h2 class="week-label"><%= current_year_and_week_number %></h2>
+    <% end %>
```

Comment:
WTF? Lets move this to the controller and test it

Reply:
Naw boooooiii. We'll make this better once we get it into production and get people starting to use it.

Additionally, we're gonna spend some time re-designing the front end very send, so this code will soon get refactored or go away entirely.